tests/integration/Core/Framework/DataAbstractionLayer/AttributeEntityIntegrationTest.php

```diff
@@ -85,6 +86,27 @@ public function testDefinitionsExist(): void
         static::assertInstanceOf(EntityRepository::class, static::getContainer()->get('attribute_entity_translation.repository'));
     }
 
+    public function testAssociationDefinitions(): void
+    {
+        $definition = static::getContainer()->get('attribute_entity_agg.definition');
+
+        static::assertInstanceOf(AttributeEntityDefinition::class, $definition);
+        static::assertNotNull($definition->getFields()->get('ownColumn'));
+        static::assertNotNull($definition->getFields()->get('attributeEntity'));
+
+        $field = $definition->getFields()->get('ownColumn');
+        static::assertInstanceOf(ManyToOneAssociationField::class, $field);
+        static::assertEquals('attribute_entity_id', $field->getStorageName());
```

Comment:
please use `assertSame` where possible instead, as it is type safe

Reply:
when comparing strings it makes no difference only on numbers

Reply:
just want to state that :slightly_smiling_face: but it would be better if we consistently using `assertSame` (objects and arrays sometimes are exceptions of course)

Reply:
okay - even when it makes no difference - i changed it 🤷‍♂️